packages/sui-studio/workbench/src/components/Root/index.js

```diff
@@ -1,24 +1,24 @@
 /* eslint import/no-webpack-loader-syntax:0 */
-import React from 'react'
-import PropTypes from 'prop-types'
+import React from "react";
```

Comment:
It seems you're not using the correct linter as simple quotes are the one we're using to format our code. Recheck it!

Reply:
Aslo there's some semicolons in the file :O

Reply:
not anymore.. 😅